From 37657daa76f390d4f1bbd9b104ea7e750556a788 Mon Sep 17 00:00:00 2001
From: auto3000 <auto3000@users.noreply.github.com>
Date: Tue, 29 Aug 2017 07:56:25 +0200
Subject: [PATCH] sun4i-codec: enable audio on nanopi-neo-air

The driver sun4i-codec relies on sun8i-codec-analog but the
latter was not correctly loaded prior the probe() of sun4i-codec.
This modification provokes the early load of sun8i-codec-analog.
---
 sound/soc/sunxi/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/sunxi/Makefile b/sound/soc/sunxi/Makefile
index fede4eb..f947050 100644
--- a/sound/soc/sunxi/Makefile
+++ b/sound/soc/sunxi/Makefile
@@ -1,6 +1,6 @@
+obj-$(CONFIG_SND_SUN8I_CODEC_ANALOG) += sun8i-codec-analog.o
 obj-$(CONFIG_SND_SUN4I_CODEC) += sun4i-codec.o
 obj-$(CONFIG_SND_SUN4I_I2S) += sun4i-i2s.o
 obj-$(CONFIG_SND_SUN4I_SPDIF) += sun4i-spdif.o
-obj-$(CONFIG_SND_SUN8I_CODEC_ANALOG) += sun8i-codec-analog.o
 obj-$(CONFIG_SND_SUN8I_CODEC) += sun8i-codec.o
 obj-$(CONFIG_SND_SUN8I_I2S) += sun8i-i2s.o
-- 
1.9.1

